demosplan/DemosPlanCoreBundle/Exception/DuplicatedTagTitleException.php

```diff
@@ -10,15 +10,33 @@
 
 namespace demosplan\DemosPlanCoreBundle\Exception;
 
+use demosplan\DemosPlanCoreBundle\Entity\Statement\TagTopic;
 use Exception;
 
 class DuplicatedTagTitleException extends Exception
 {
+    public function __construct(string $message,
+                                protected readonly TagTopic $topic,
+                                protected readonly string $tagTitle,
+    ) {
+        parent::__construct($message, 0, null);
```

Comment:
`0, null` is not needed as they are the default values

Reply:
done thx https://github.com/demos-europe/demosplan-core/pull/2018/commits/98ea6dc40e8747273304f891b14de3eafe0dc9e0